src/Row.cpp

```diff
@@ -13,7 +13,7 @@ Row::Row(
     m_values.reserve(field_count);
     for (size_t i = 0; i < field_count; ++i) {
         auto* mysql_value = mysql_row[i];
-        std::string_view data {};
+        std::optional<std::string_view> data;
```

Comment:
can data be std::move()'ed into m_values?  Probably about the same cost as a copy but the intent is there